src/rmt.rs

```diff
@@ -856,3 +915,189 @@ mod chip {
         }
     }
 }
+
+use self::config::ReceiveConfig;
+/// The RMT receiver.
+///
+/// Use [`RxRmtDriver::start()`] to receive pulses.
+///
+/// See the [rmt module][crate::rmt] for more information.
+use alloc::vec::Vec;
+use core::convert::TryInto;
+use core::ptr;
+
+pub struct RxRmtDriver<'d> {
+    channel: u8,
+    _p: PhantomData<&'d mut ()>,
+    pub pulse_pair_vec: Vec<PulsePair>,
```

Comment:
Why should that be a member at all? Remove. (And see below for an alternative suggestion.)